Remove path prefix so it can find these tools from buildhost or native packages

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
Index: qt-creator-opensource-src-4.5.1/share/qtcreator/translations/translations.pro
===================================================================
--- qt-creator-opensource-src-4.5.1.orig/share/qtcreator/translations/translations.pro
+++ qt-creator-opensource-src-4.5.1/share/qtcreator/translations/translations.pro
@@ -12,10 +12,10 @@ defineReplace(prependAll) {
     return($$result)
 }
 
-XMLPATTERNS = $$shell_path($$[QT_INSTALL_BINS]/xmlpatterns)
-LUPDATE = $$shell_path($$[QT_INSTALL_BINS]/lupdate) -locations relative -no-ui-lines -no-sort
-LRELEASE = $$shell_path($$[QT_INSTALL_BINS]/lrelease)
-LCONVERT = $$shell_path($$[QT_INSTALL_BINS]/lconvert)
+XMLPATTERNS = $$OE_QMAKE_PATH_EXTERNAL_HOST_BINS/xmlpatterns
+LUPDATE = $$OE_QMAKE_PATH_EXTERNAL_HOST_BINS/lupdate -locations relative -no-ui-lines -no-sort
+LRELEASE = $$OE_QMAKE_PATH_EXTERNAL_HOST_BINS/lrelease
+LCONVERT = $$OE_QMAKE_PATH_EXTERNAL_HOST_BINS/lconvert
 
 wd = $$replace(IDE_SOURCE_TREE, /, $$QMAKE_DIR_SEP)
 
